site/components/pages/Retirements/List/index.tsx

```diff
@@ -11,10 +11,11 @@ import * as styles from "./styles";
 
 type Props = {
   klimaRetires: KlimaRetire[];
+  domain?: string;
 };
 
 export const AllRetirements: FC<Props> = (props) => {
-  const { klimaRetires } = props;
+  const { klimaRetires, domain } = props;
```

Comment:
```suggestion
  const { klimaRetires, nameserviceDomain } = props;
```

just to be very clear so noone gets confused and thinks this is the actual web domain :)

Reply:
Done